fendermint/vm/interpreter/src/fvm/state/ipc.rs

```diff
@@ -75,11 +77,33 @@ impl<DB: Blockstore> GatewayCaller<DB> {
         self.getter.call(state, |c| c.bottom_up_check_period())
     }
 
+    /// Fetch the bottom-up messages enqueued in a given checkpoint.
+    pub fn bottom_up_msgs(
+        &self,
+        state: &mut FvmExecState<DB>,
+        height: u64,
+    ) -> anyhow::Result<Vec<getter::CrossMsg>> {
+        // NOTE: This particular method is marked as deprecated but it's currently the only known way to retrieve
```

Comment:
Why is this method deprecated (//cc @dnkolegov)? I would say that this is the right way to get the bottom-up cross-net messages. 
* When there is a new bottomup message, [`commitBottomUpMsg`](https://github.com/consensus-shipyard/ipc-solidity-actors/blob/bbf1d9b2bf7250a0c5432894e15ed5f4b0453692/src/lib/LibGateway.sol#L178) pushes the message to the height of the next checkpoint to be created.
* When we need to populate that checkpoint we can directly get the bottomup messages as you are doing.

Reply:
It was deprecated because it returned `BottomUpMessageLegacy` several days ago.
Now, it looks like it's usual - https://github.com/consensus-shipyard/ipc-solidity-actors/blob/bbf1d9b2bf7250a0c5432894e15ed5f4b0453692/src/gateway/GatewayGetterFacet.sol#L58

Reply:
Ah, yes, I should have removed this comment, thanks!